client/components/auth/ForgotPassword.vue

```diff
@@ -7,35 +7,36 @@
       class="mb-5">
       {{ error || 'Sending reset email...' }}
     </v-alert>
-    <form v-if="!error" @submit.prevent="submit">
-      <v-text-field
-        v-model="email"
-        v-validate="{ required: true, email: true }"
-        :error-messages="vErrors.collect('email')"
-        type="email"
-        name="email"
-        label="Email"
-        placeholder="Email"
-        prepend-inner-icon="mdi-email-outline"
-        outlined />
-      <div class="d-flex">
-        <v-btn
-          @click="$router.go(-1)"
-          tag="a"
-          text
-          class="px-1">
-          <v-icon>mdi-chevron-left</v-icon>
-          Back
-        </v-btn>
-        <v-spacer />
-        <v-btn
-          :disabled="!isValid || showMessage"
-          type="submit"
-          color="primary darken-1">
-          Send reset email
-        </v-btn>
-      </div>
-    </form>
+    <validation-observer v-if="!error" v-slot="{ invalid }">
+      <form @submit.prevent="submit">
```

Comment:
Also for all form of this type, you should use `handleSubmit` [scoped prop ](https://logaretm.github.io/vee-validate/api/validation-observer.html#rendering) to prevent `submit` method from triggering. [It will only trigger if the form is valid. ](https://logaretm.github.io/vee-validate/guide/forms.html#validate-before-submit)